ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml

```diff
@@ -242,18 +361,38 @@
                     current-parameters: true
                   - name: 'ceph-ansible-prs-dev-lvm_batch_container'
                     current-parameters: true
-
       - conditional-step:
           condition-kind: shell
           condition-command: |
             #!/bin/bash
             set -x
-            # if the current branch is stable-3.0 or stable-3.1 then we don't run these tests
-            if [[ "$ghprbTargetBranch" == "stable-3.1" ]] || [[ "$ghprbTargetBranch" == "stable-3.0" ]]; then
+            # if the target branch is stable-3.2 then we RUN these tests.
```

Comment:
you need to apply the same condition to add_osds

Reply:
we run the test only if the branch is stable-3.2, see L374

Reply:
that's what I'm saying, we need the same thing for add_osd